src/deadline/job_attachments/api/manifest.py

```diff
@@ -247,3 +260,182 @@ def process_output(status: FileStatus, path: str, output_diff: ManifestDiff):
             process_output(fast_diff_item[1], fast_diff_item[0], output)
 
     return output
+
+
+def _manifest_upload(
+    manifest_file: str,
+    s3_bucket_name: str,
+    manifest_path: str,
+    s3_metadata: Dict[str, Any],
+    boto_session: boto3.Session,
+    logger: ClickLogger = ClickLogger(False),
+):
+    """
+    BETA API - This API is still evolving but will be made public in the near future.
+    API to upload a job attachment manifest to the Content Address Storage. Manifests will be
+    uploaded to s3://{s3_bucket_name}/Manifest/{manifest_path} as per the Deadline CAS folder structure.
+    manifest_file: File Path to the manifest file for upload.
+    s3_bucket_name: S3 bucket name.
+    manifest_path: S3 path for the manifest.
+    s3_metadata: Additional S3 file metadata tagged on upload.
```

Comment:
This is inconsistent with the command, I found it awkward from AWS CLI/SDK perspective since they're always consistent. Thoughts?

Reply:
What do you mean? Will check with you offline

Reply:
I meant this s3_metadata is not available via the command, while required for the API. I thought our plan is to have feature parity between the CLI and API, is that no longer true?

Reply:
Got it - yes I will move s3_metadata tagging from the CLI into the API.